src/insexpand.c

```diff
@@ -924,7 +924,13 @@ ins_compl_insert_bytes(char_u *p, int len)
     int
 ins_compl_col_range_attr(int col)
 {
-    if (col >= compl_col && col < compl_ins_end_col)
+    int	    scol = compl_col;
+    if ((get_cot_flags() & COT_FUZZY))
+	return -1;
```

Comment:
Doesn't this mean `ComplMatchIns` is always disabled with fuzzy completion?

Reply:
currently it is.. can be supported ...